Assets/MixedRealityToolkit/Definitions/Diagnostics/MixedRealityDiagnosticsProfile.cs

```diff
@@ -35,12 +35,47 @@ public class MixedRealityDiagnosticsProfile : BaseMixedRealityProfile
 
         [SerializeField]
         [Tooltip("The amount of time, in seconds, to collect frames for frame rate calculation.")]
-        [Range(0.01f, 1.0f)]
-        private float frameRateDuration = 0.1f;
+        private float frameSampleRate = 0.1f;
```

Comment:
why remove the range? what would happen if the value were 0 or negative?

Reply:
also, can you add a previouslyserializedas attribute to support legacy profiles?

Reply:
You mean `FormerlySerializedAs` ?

Reply:
Great idea about backwards compat with FormerlySerializedAs. I'll add that. As for the range question. I didn't think it made sense to constrain users to the 0-1 range. Since they may want to sample every 2 seconds and so on. If the sample rate is <= 0 then the frame rate will just be sampled every frame.